<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="paintFrame">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>448</width>
      <height>448</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="brushToolButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Brush</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipToolButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
   <widget class="QListWidget" name="framesListWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="iconSize">
     <size>
      <width>75</width>
      <height>75</height>
     </size>
    </property>
    <property name="flow">
     <enum>QListView::LeftToRight</enum>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteFrameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="duplicateFrameButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>431</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorsButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Colors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redoButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Redo</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>831</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
